packages/product/src/services/product-module-service.ts

```diff
@@ -369,6 +373,145 @@ export default class ProductModuleService<
     return JSON.parse(JSON.stringify(products))
   }
 
+  // TODO: cleanup, abstract common create and update steps
```

Comment:
comment: This would be great!

suggestion: Add todo to include categories + collection

Reply:
Ah, good catch. I'll add the categories and collections now. Will do the cleanup in a follow up PR since it will end up being very opinionated.

Reply:
👍